### PR TITLE
Set dashboard url from server during login flow

### PR DESCRIPTION
**Pull request by Piccirello**
- Set dashboard url from server during login flow
- Sort cli configuration before printing
- Replace duplication with iteration

### CODE REVIEW

File: pkg/configuration/config.go [hunk 1]
```diff
@@ -324,25 +290,17 @@ func parseScope(scope string) (string, error) {
 
 // IsValidConfigOption whether the specified key is a valid option
 func IsValidConfigOption(key string) bool {
-	return key == "token" || key == "project" || key == "config" || key == "api-host" || key == "verify-tls"
+	return key == "token" || key == "project" || key == "config" || key == "api-host" ||
```
Comment (bvallelunga) [inline pkg/configuration/config.go]: Can we put this those strings in a set and check if the key exists in the set? I think that would be much cleaner and easier to extend in the future.

File: pkg/utils/print.go [hunk 2]
```diff
@@ -371,23 +343,26 @@ func PrintConfigs(configs map[string]models.Config, jsonFlag bool) {
...
+	// sort by scope, then by name
+	sort.Slice(rows, func(a, b int) bool {
+		if rows[a][2] > rows[b][2] {
+			return false
 		}
-		if config.VerifyTLS != "" {
-			rows = append(rows, []string{"verify-tls", config.VerifyTLS, scope})
+		if rows[a][2] < rows[b][2] {
+			return true
 		}
```
Comment (bvallelunga) [inline pkg/utils/print.go]: I think we can clean up this code with

``` go
if rows[a][2] != rows[b][2] {
    return rows[a][2] < rows[b][2] 
}
```

**Review by bvallelunga — APPROVED** [review]

File: pkg/utils/print.go (hunk 2 above)
Comment (Piccirello) [reply pkg/utils/print.go]: +1

File: pkg/configuration/config.go (hunk 1 above)
Comment (Piccirello) [reply pkg/configuration/config.go]: This is the equivalent code in Go. I don't think it's worth it:

```golang
	configOptions := [...]string{
		"token",
		"project",
		"config",
		"api-host",
		"dashboard-host",
		"verify-tls",
	}

	for _, option := range configOptions {
		if key == option {
			return true
		}
	}

	return false
```

Comment (bvallelunga) [reply pkg/configuration/config.go]: What about:

``` go
configOptions := [...]string{
		"token",
		"project",
		"config",
		"api-host",
		"dashboard-host",
		"verify-tls",
}

val, exists := dict["foo"]
return exists
```

Reference
https://stackoverflow.com/questions/2050391/how-to-check-if-a-map-contains-a-key-in-go

Comment (Piccirello) [reply pkg/configuration/config.go]: +1

**Pull request merged**